test/Mono.Linker.Tests.Cases/RequiresCapability/RequiresOnClass.cs

```diff
@@ -544,10 +555,12 @@ static void TestDAMAccess ()
 				typeof (DerivedWithRequiresOnTypeOverBaseWithNoRequires).RequiresPublicMethods ();
```

Comment:
Do we need to update these comments? Looks like at least BaseWithRequiresOnType is warning now.